__tests__/app.test.js

```diff
@@ -31,31 +31,91 @@ describe('app routes', () => {
       return client.end(done);
     });
 
-    test('returns animals', async() => {
+    test('returns todos', async() => {
 
       const expectation = [
         {
           'id': 1,
-          'name': 'bessie',
-          'coolfactor': 3,
+          'todo': 'wash the dishes',
+          'completed': false,
           'owner_id': 1
-        },
+        },  
         {
           'id': 2,
-          'name': 'jumpy',
-          'coolfactor': 4,
+          'todo': 'clean the bathroom',
+          'completed': false,
           'owner_id': 1
         },
         {
           'id': 3,
-          'name': 'spot',
-          'coolfactor': 10,
+          'todo': 'fold the laundry',
+          'completed': false,
+          'owner_id': 1
+        },
+        {
+          'id': 4,
+          'todo': 'take out the trash',
+          'completed': false,
+          'owner_id': 1
+        },
+        {
+          'id': 5,
+          'todo': 'stretch',
+          'completed': false,
+          'owner_id': 1
+        },
+        {
+          'id': 6,
+          'todo': 'do my homework',
+          'completed': false,
           'owner_id': 1
         }
       ];
 
       const data = await fakeRequest(app)
-        .get('/animals')
+        .get('/todos')
```

Comment:
if you wanted to go with the suggestion I made: change the expectation for only one user, change this to /api/todos, set the token just like the tests below.